src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_Conversion.cs

```diff
@@ -1262,7 +1262,7 @@ public static SpecialMember GetIntPtrConversionMethod(TypeSymbol source, TypeSym
             }
             else if (t0Type == SpecialType.System_UIntPtr)
             {
-                if (source.TypeKind == TypeKind.Pointer)
+                if (source.IsPointerOrFunctionPointer())
```

Comment:
>source.IsPointerOrFunctionPointer() [](start = 20, length = 35)

Are changes in this file covered by tests?

Reply:
See the changes in this test: https://github.com/dotnet/roslyn/pull/43021/files#diff-970e57f3956cd201f61fe1f537e5c02cR233-R272